### DISCUSSION TITLE
Is the Bot Crafter still necessary?

### DISCUSSION THREAD

**Discussion by LofiMonk** [LofiMonk/bot-topia#6] [General]
With Mojang's new updates to Minecraft and the addon API it is now possible to have custom items with custom recipes show up in the crafting bench. This was not possible previously and lead me to creating the Bot Crafter.
   Before the new changes you had to memorize recipes. I didn't bother adding recipes to every item because it would be too much for anyone to remember.  That is why the Bot crafter now shows up in the crafting bench menu, it was one of the items I did make a recipe for.
   I also believe the Bot Crafter is helpful for players that are new to the addon. I think it helps guide new players in what order they should craft items in to make the bots. 

**TLDR**  Since we can now put items into the regular crafting bench should the Bot Crafter be done away with?
